packages/runtime/container-runtime/src/opLifecycle/README.md

```diff
@@ -157,86 +133,7 @@ In general, compression offers a trade-off between higher compute costs, lower b
 
 ## How it works
```

Comment:
I switched the order of the two "How it works" sections since Grouped Batching is the default now.

Reply:
I'm not sure if what is by default matters - all these options are On by default.

Reply:
That said, I'd recommend to merge compression & op grouping into one flag, and get rid below of any reference that talks about "empty ops"

Reply:
AB#8124